src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -11027,13 +10986,15 @@ private ExpressionSyntax ParseExpressionContinued(ExpressionSyntax leftOperand,
                 // We either have a binary or assignment operator here, or we're finished.
                 var tk = this.CurrentToken.ContextualKind;
 
-                bool isAssignmentOperator = false;
+                var isAssignmentOperator = false;
```

Comment:
Let's avoid unless there's a reason to change
```suggestion
                bool isAssignmentOperator = false;
```